src/GiasDataLoader/CommandLineOptions.cs

```diff
@@ -0,0 +1,19 @@
+using CommandLine;
```

Comment:
A nice lib. 👍

Reply:
Yeah, I like it when I need to do CLI things like this